app/client/cli/utils.go

```diff
@@ -147,8 +152,15 @@ func postRawTx(ctx context.Context, pk crypto.Ed25519PrivateKey, j []byte) (*rpc
 }
 
 func getNonce() string {
-	rand.Seed(time.Now().UTC().UnixNano())
-	return fmt.Sprintf("%d", rand.Uint64())
+	max := new(big.Int)
```

Comment:
There's probably an opportunity here to move random number generation (with fallback) into `shared`. Potentially `shared/crypto/rand.go`?